modules/ossm-integrating-with-user-workload-monitoring.adoc

```diff
@@ -0,0 +1,176 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/ossm-observability.adoc
+////
+
+:_content-type: PROCEDURE
+[id="ossm-integrating-with-user-workload-monitoring_{context}"]
+= Integrating with user-workload monitoring
+
+By default, OpenShift Service Mesh (OSSM) installs the Service Mesh Control Plane (SMCP) with a dedicated instance of Prometheus for collecting metrics from a mesh. However, production systems need more advanced monitoring systems, like OpenShift Monitoring for user-defined projects.
+
+The following steps show how to integrate Service Mesh with User-Workload Monitoring.
+
+.Procedure
+
+. Enable monitoring in user-defined projects by adding the following configuration to the `cluster-monitoring-config` config map:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true
+----
+
+. Complete one of the following options to configure the Service Mesh control plane for User-Workload Monitoring.
```

Comment:
"Complete one of the following options" - this is incorrect.
1) Disable Prometheus addon - required.
2) Configure extension provider - required.
3) Disable network policy **OR** apply custom network policy to allow ingress traffic from user-workload monitoring namespace.